frontend/src/Store/Actions/blocklistActions.js

```diff
@@ -83,63 +83,63 @@ export const defaultState = {
 };
 
 export const persistState = [
-  'blacklist.pageSize',
-  'blacklist.sortKey',
-  'blacklist.sortDirection',
-  'blacklist.columns'
+  'blocklist.pageSize',
+  'blocklist.sortKey',
+  'blocklist.sortDirection',
+  'blocklist.columns'
```

Comment:
Prob needs frontend cache migration

Reply:
And how would we do that...

Reply:
Similar to this 

https://github.com/Sonarr/Sonarr/blob/develop/frontend/src/Store/Migrators/migrateAddSeriesDefaults.js

Not a huge deal but, maybe handy if it's easy to avoid killing all preferences for that screen

Reply:
Looking at that over the next couple of days - just getting a few bits done around the house